index.html

```diff
@@ -12,7 +12,71 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <nav class="top-navigation">
+        <div class="content">
+            <ul class="links">
+                <li class="active">Meet Karma</li>
+                <li>How it Works</li>
+                <li>Store</li>
+                <li>Blog</li>
+                <li>Help</li>
+                <li>Login</li>
+            </ul>
+            <img class="logo"  src="img/karma-logo.svg" alt="Logo Image" width="20"/>
+        </div>
+    </nav>
+    <section class="section-1">
+        <div class="content">
+            <h1>Introducing Karma</h1>
+            <h2>Bring WiFi with you, everywhere you go.</h2>
+            <div class="buttons">
+                <button>Learn More</button>
+            </div>
+        </div>
+    </section>
+    <section class="section-2">
```

Comment:
Avoid naming elements by order of their appearance on the page, as this may change when you switch from mobile view to desktop view. It also doesn't give much indication of what is inside the section - try and be a little more descriptive.